subsys/shell/Kconfig

```diff
@@ -111,8 +111,15 @@ config SHELL_START_OBSCURED
 	  If enabled, don't echo actual character, but echo * instead.
 	  This is used for login prompts.
 
+config SHELL_VT100_COMMANDS
+	bool "Enable VT100 commands in shell"
+	default y if !SHELL_MINIMAL
```

Comment:
I'm not convinced that we want to deprive Shell's minimal version of VT100 support. Apart from that, I am fine with the change.

Reply:
I wasn't sure what the default option should be but in that case I will change that so it will be enabled by default even if shell minimal is set.

Reply:
I mean I would like to have VT100 commands enabled by default in minimal and not minimal configuration.

Reply:
Already changed.